orbit/cmd/orbit/orbit.go

```diff
@@ -351,10 +350,8 @@ func main() {
 		g.Add(r.Execute, r.Interrupt)
 
 		// Extension tables not yet supported on Windows.
```

Comment:
I can only see extension tables for `macOS` and `Windows` now, not for linux, should `linux` be excluded here? (I know we have a stub but just to be extra clear about current supported extension tables).

Reply:
Ah, disregard, I can see that, by default, now it's registering `puppet_info`, `puppet_logs`, etc., I'm guessing those are multi-platform.